packages/rum-recorder/src/boot/recorder.spec.ts

```diff
@@ -39,22 +40,8 @@ describe('startRecording', () => {
       )
 
     const requestSendSpy = spyOn(HttpRequest.prototype, 'send')
-
-    waitRequests = (expectedRequestCount, callback) => {
-      const requests: Array<{ data: FormData; size: number }> = []
-      requestSendSpy.and.callFake((data, size) => {
-        if (requests.push({ size, data: data as FormData }) === expectedRequestCount) {
-          callback(requests)
-        }
-      })
-    }
-
-    expectNoExtraRequest = (done) => {
-      requestSendSpy.and.callFake(() => {
-        fail('Unexpected request received')
-      })
-      setTimeout(done, 300)
-    }
+    expectNoExtraRequestSendCalls = expectNoExtraCall.bind(null, requestSendSpy)
+    waitRequestSendCalls = waitCalls.bind(null, requestSendSpy)
```

Comment:
Since these kinds of binds should be needed for every usage of this utility, what about having a single API that returns the two functions bounded?
something like:
```ts
const { waitCalls, expectNoExtraCall } = foo(spyOn(HttpRequest.prototype, 'send'))
```